src/lando/api/tests/test_landings.py

```diff
@@ -935,8 +886,8 @@ def test_format_patch_fail(
 
     revisions = [
         create_patch_revision(1, patch=PATCH_FORMATTING_PATTERN_FAIL),
-        create_patch_revision(2),
-        create_patch_revision(3),
+        create_patch_revision(2, patch=normal_patch(0)),
+        create_patch_revision(3, patch=normal_patch(1)),
```

Comment:
Without an explicit `patch`, `NORMAL_PATCH_1` will always get used. Hg doesn't mind reapplying an existing patch, and just skips it silently, but Git is not so lenient, and fails.

Making sure the patches are different and part of the same series makes both SCMs happy for this test.